### PR TITLE
Implement website service by using ETCD

### PR DESCRIPTION
**Pull request by lidongze0629**
### 👀 PR includes

✨ Feature

- [x] Implement website service by using ETCD
  - [x] Case Interface
  - [x] Project Interface
  - [ ] Save Interface (createShare interface not found)

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌     | ✅    |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/gi-site/.umirc.ts [hunk 1]
```diff
@@ -128,6 +128,12 @@ export default {
       ],
     },
   ],
+  proxy: {
```
Comment (pomelo-nwu) [inline packages/gi-site/.umirc.ts:131]: 这个很危险，因为如果访问的是内网服务器，就不能走这个代理，先这样走跑通，后面我处理下

File: packages/gi-site/docker/website.Dockerfile [hunk 2]
```diff
@@ -1,25 +1,30 @@
 # Website for graphinsight
 
-######################## BUILDER: Website ##########################
-# from docker.io/library/node:16.17.1 AS builder
-from python:3.9-alpine AS builder
+from docker.io/library/node:16
 
-COPY ./dist /work/dist
+COPY . /root/G6VP
```
Comment (yecol) [inline packages/gi-site/docker/website.Dockerfile:5]: if possible, do not copy to root. /workspace or /work would be better.

File: packages/gi-site/docker/website.Dockerfile [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
-# # RUN npm install -g pnpm
+# install pnpm
+RUN npm install -g pnpm
 
-# # RUN cd /work/G6VP/packages/gi-site && \
-# #   pnpm install && \
-# #   npm run build
+# download etcd
+# see deatil in gi-httpservice/app/service/graphinsight/utils.ts
+RUN cd /root && \
```
Comment (yecol) [inline packages/gi-site/docker/website.Dockerfile:15]: why it should be /root?

File: packages/gi-site/docker/docker-entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+# listen on 127.0.0.1:7001 default
+cd ${G6VP_DIR}/packages/gi-httpservice
+mkdir -p ${RUNTIME_DIR}/gihttpservice
+npm run dev &> ${RUNTIME_DIR}/gihttpservice/output.log &
+
+# gi site
+# listen on 127.0.0.1:8000 default
+cd ${G6VP_DIR}/packages/gi-site
+npm run start
+
```
Comment (yecol) [inline packages/gi-site/docker/docker-entrypoint.sh:28]: line 1-8, line-13, line-21, move to dockerfile?
GI site, which as the main entrance, should be host on port 80?

**Pull request merged**